src/core/include/openvino/op/constant.hpp

```diff
@@ -161,6 +161,9 @@ class OPENVINO_API Constant : public Op {
         case Type_t::nf4:
             fill_data<Type_t::nf4>(value);
             break;
+        case Type_t::string:
+            fill_data<Type_t::string>(value);
```

Comment:
how string constants are supposed to be stored in IR file?
I mean that here we stored only "pointers", while an IR file has to store actual data as well

Reply:
This release we don't plan to serialize such constants in IR. We have no critical models with string constants